_topic_map.yml

```diff
@@ -474,9 +474,9 @@ Topics:
 #    File: configuring-registry-storage-openstack-user-infrastructure
 #  - Name: Configuring registry storage for Azure user-provisioned infrastructure
 #    File: configuring-registry-storage-azure-user-infrastructure
-  - Name: Configuring registry storage for bare metal
+  - Name: Configuring the registry for bare metal
```

Comment:
Should you also update AWS and GCP title from "registry storage" to "the registry" for consistency?

Reply:
I thought about that too, but they're currently focused on setting up  registry storage. I suppose that could change so I'll update them for consistency.

Reply:
Updated.